### DISCUSSION TITLE
The free energy of an alloy system

### DISCUSSION THREAD

**Discussion by Liang1Kai** [ftherrien/VaspGibbs#8] [VaspGibbs Forum]
Hi Developer,
Could this program be used to calculate the free energy of an alloy system?In alloy system, there is no need to calculate the free energy of translation and rotation. Can the free energy of these two parts not be calculated?
Looking forward to reply.
Sincerely！

**Comment by ftherrien** [comment]
Hi @Liang1Kai,

To calculate the free energy of an alloy, you probably need the configuration entropy and since you have a crystal system you will also need the phonon DOS for which you will need to do actual phonon calculations (not only at gamma). This is something I would love to implement in the future, but haven't gotten the chance to do.

If you don't use the `--molecule` flag the translation and rotation are *not* calculated. If you make a large supercell and if your alloy does not have many possible configurations (i.e. there is only one way to organize the atoms in the cell) you might get away with using VaspGibbs as is. 

This paper might be helpful to you: https://www.frontiersin.org/articles/10.3389/fchem.2020.00757/full

Good luck!

**Comment by Liang1Kai** [comment]
Thank you very much for your reply, which is very helpful to me.
have a good time！
thanks again！



---- Replied Message ----
| From | Félix ***@***.***> |
| Date | 01/31/2024 05:51 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [ftherrien/VaspGibbs] The free energy of an alloy system (Discussion #8) |

Hi @Liang1Kai,

To calculate the free energy of an alloy, you probably need the configuration entropy and since you have a crystal system you will also need the phonon DOS for which you will need to do actual phonon calculations (not only at gamma). This is something I would love to implement in the future, but haven't gotten the chance to do.

If you don't use the --molecule flag the translation and rotation are not calculated. If you make a large supercell and if your alloy does not have many possible configurations (i.e. there is only one way to organize the atoms in the cell) you might get away with using VaspGibbs as is.

This paper might be helpful to you: https://www.frontiersin.org/articles/10.3389/fchem.2020.00757/full

Good luck!
